client/my-sites/plugins/plugin-navbar-footer/index.tsx

```diff
@@ -0,0 +1,389 @@
+	);
+
+	return (
+		<>
+			<section
+				id="lpc-footer-nav"
+				data-vars-ev-id="lpc-footer-nav"
+				className="lpc lpc-footer-nav"
+				data-vars-ev-classname="lpc lpc-footer-nav"
+			>
+				<h2 className="lp-hidden">WordPress.com</h2>
+				<div className="lpc-footer-nav-wrapper">
+					<div className="lpc-footer-nav-container">
+						<div>
+							<h3>{ translate( 'Products' ) }</h3>
+							<ul>
+								<li>
+									<a href="https://wordpress.com/hosting/">{ translate( 'WordPress Hosting' ) }</a>
+								</li>
+								<li>
+									<a href="https://wordpress.com/domains/">{ translate( 'Domain Names' ) }</a>
+								</li>
+								<li>
+									<a href="https://wordpress.com/website-builder/">
+										{ translate( 'Website Builder' ) }
+									</a>
+								</li>
+								<li>
+									<a href="https://wordpress.com/create-blog/">{ translate( 'Create a Blog' ) }</a>
+								</li>
+								<li>
+									<a href="https://wordpress.com/professional-email/">
+										{ translate( 'Professional Email' ) }
+									</a>
+								</li>
+								<li>
+									<a href="https://wordpress.com/p2/?ref=wpcom-product-menu">
+										{ translate( 'P2: WordPress for Teams' ) }
+									</a>
+								</li>
+								<li>
+									<a href="https://wpvip.com/" data-is_external="1">
+										{ translate( 'Enterprise' ) }{ ' ' }
+										<span className="lp-link-chevron-external">{ translate( 'Solutions' ) }</span>
+									</a>
+								</li>
+								<li>
+									<a
+										href="https://wordpress.com/do-it-for-me/?ref=footer_pricing"
+										title="WordPress Website Building Service"
+									>
+										{ translate( 'Built by&nbsp;WordPress.com' ) }
+									</a>
+								</li>
+							</ul>
+						</div>
+						<div>
+							<h3>{ translate( 'Features' ) }</h3>
+							<ul>
+								<li>
+									<a href="https://wordpress.com/features/" title="WordPress.com Features">
+										{ translate( 'Overview' ) }
+									</a>
+								</li>
+								<li>
+									<a href="https://wordpress.com/themes">{ translate( 'WordPress Themes' ) }</a>
+								</li>
+								<li>
+									<a href="https://wordpress.com/install-plugins/">
+										{ translate( 'WordPress Plugins' ) }
+									</a>
+								</li>
+								<li>
+									<a href="https://wordpress.com/google/">{ translate( 'Google Apps' ) }</a>
+								</li>
+							</ul>
+						</div>
+						<div>
+							<h3>{ translate( 'Resources' ) }</h3>
+							<ul>
+								<li>
+									<a href="https://wordpress.com/support/">
+										{ translate( 'WordPress.com Support' ) }
+									</a>
+								</li>
+								<li>
+									<a href="https://wordpress.com/forums/">{ translate( 'WordPress Forums' ) }</a>
+								</li>
+								<li>
+									<a href="https://wordpress.com/blog/">{ translate( 'WordPress News' ) }</a>
+								</li>
+								<li>
+									<a href="https://wordpress.com/go/">
+										{ translate( 'Website Building&nbsp;Tips' ) }
```

Comment:
`Website Building&nbsp;Tips` and `Built by&nbsp;WordPress.com` seem output the `&nbsp;` on the page.
<img width="240" alt="Screenshot on 2022-09-16 at 11-31-34" src="https://user-images.githubusercontent.com/2749938/190594490-a8a15f7b-7a1e-4862-b4f3-a8572e9ab225.png">